pages/places/[id].js

```diff
@@ -44,6 +50,10 @@ export default function Place({ getUniqueValues }) {
           toggleFavorite={toggleFavorite}
           place={place}
           getUniqueValues={getUniqueValues}
+          isImage={isImage}
+          setIsImage={setIsImage}
+          publicId={publicId}
+          setPublicId={setPublicId}
```

Comment:
As mentioned before: Try to avoid  passing state setter functions as props. It's better to encapsulate them within handler functions.